src/Extension.php

```diff
@@ -208,336 +208,99 @@ public static function blocks_payment_method_type_registration( PaymentMethodReg
+			PaymentMethods::APPLE_PAY               => 'pronamic_pay_apple_pay',
+			PaymentMethods::BANCONTACT              => 'pronamic_pay_mister_cash',
+			PaymentMethods::BANK_TRANSFER           => 'pronamic_pay_bank_transfer',
+			PaymentMethods::BELFIUS                 => 'pronamic_pay_belfius',
+			PaymentMethods::BILLIE                  => 'pronamic_pay_billie',
+			PaymentMethods::BITCOIN                 => 'pronamic_pay_bitcoin',
+			PaymentMethods::BLIK                    => 'pronamic_pay_blik',
+			PaymentMethods::BUNQ                    => 'pronamic_pay_bunq',
+			PaymentMethods::CARD                    => 'pronamic_pay_card',
+			PaymentMethods::CREDIT_CARD             => 'pronamic_pay_credit_card',
+			PaymentMethods::DIRECT_DEBIT            => 'pronamic_pay_direct_debit',
+			PaymentMethods::DIRECT_DEBIT_BANCONTACT => 'pronamic_pay_direct_debit_bancontact',
+			PaymentMethods::DIRECT_DEBIT_IDEAL      => 'pronamic_pay_direct_debit_ideal',
+			PaymentMethods::DIRECT_DEBIT_SOFORT     => 'pronamic_pay_direct_debit_sofort',
+			PaymentMethods::FOCUM                   => 'pronamic_pay_focum',
+			PaymentMethods::EPS                     => 'pronamic_pay_eps',
+			PaymentMethods::GIROPAY                 => 'pronamic_pay_giropay',
+			PaymentMethods::GOOGLE_PAY              => 'pronamic_pay_google_pay',
+			PaymentMethods::IDEAL                   => 'pronamic_pay_ideal',
+			PaymentMethods::IDEALQR                 => 'pronamic_pay_idealqr',
+			PaymentMethods::IN3                     => 'pronamic_pay_in3',
+			PaymentMethods::KBC                     => 'pronamic_pay_kbc',
+			PaymentMethods::KLARNA_PAY_LATER        => 'pronamic_pay_klarna_pay_later',
+			PaymentMethods::KLARNA_PAY_NOW          => 'pronamic_pay_klarna_pay_now',
+			PaymentMethods::KLARNA_PAY_OVER_TIME    => 'pronamic_pay_klarna_pay_over_time',
+			PaymentMethods::MAESTRO                 => 'pronamic_pay_maestro',
+			PaymentMethods::MASTERCARD              => 'pronamic_pay_mastercard',
+			PaymentMethods::MB_WAY                  => 'pronamic_pay_mb_way',
+			PaymentMethods::MOBILEPAY               => 'pronamic_pay_mobilepay',
+			PaymentMethods::PAYCONIQ                => 'pronamic_pay_payconiq',
+			PaymentMethods::PAYPAL                  => 'pronamic_pay_paypal',
+			PaymentMethods::PRZELEWY24              => 'pronamic_pay_przelewy24',
+			PaymentMethods::RIVERTY                 => 'pronamic_pay_riverty',
+			PaymentMethods::SANTANDER               => 'pronamic_pay_santander',
+			PaymentMethods::SOFORT                  => 'pronamic_pay_sofort',
+			PaymentMethods::SPRAYPAY                => 'pronamic_pay_spraypay',
+			PaymentMethods::SWISH                   => 'pronamic_pay_swish',
+			PaymentMethods::TWINT                   => 'pronamic_pay_twint',
+			PaymentMethods::V_PAY                   => 'pronamic_pay_v_pay',
+			PaymentMethods::VIPPS                   => 'pronamic_pay_vipps',
+			PaymentMethods::VISA                    => 'pronamic_pay_visa',
+		];
```

Comment:
This can be further simplified, as only exceptions to the `'pronamic_pay_' . $payment_method->get_id()` format have to be mapped explicitly for the correct `$woo_id`.

Reply:
Simplified in 9a35cda0ec582db97f52dee447ca41a284562194.